### PR TITLE
updated changelog with changes from elasticsearch 6.0.1 upgrade

### PR DESCRIPTION
**Pull request by chaudum**

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -15,9 +15,40 @@ the appropriate section of the docs.
 Breaking Changes
 ================
 
+- Dropped support for tables that have been created with CrateDB prior to
+  version 2.0. When upgrading, you need to re-index your data. This can be done
```
[the remark this answers is not in the document]
Comment (andreidan) [reply CHANGES.txt]: Should we use a different term than "re-index" here (as that has a different specific meaning in the db world )? Maybe say one needs to snapshot and reload the data?

[the remark this answers is not in the document]
Comment (matriv) [reply CHANGES.txt]: Yep totally! That's why the syscheck says "table recreation"

[the remark this answers is not in the document]
Comment (chaudum) [reply CHANGES.txt]: good points! I think we also need to add upgrade notes to the changes, like we did for the 2.0 release, where we describe more in-depth what needs to be done

File: CHANGES.txt [hunk 2]
```diff
@@ -15,9 +15,45 @@ the appropriate section of the docs.
 Breaking Changes
 ================
 
+- Dropped support for tables that have been created with CrateDB prior to
+  version 2.0. Tables which require upgrading are indicated in the cluster
+  checks, including visually shown in the Admin UI, if running the latest 2.2
+  or 2.3 release. The upgrade of tables needs to happend before updating
```
Comment (mxm) [inline CHANGES.txt]: happen

File: CHANGES.txt [hunk 3]
```diff
@@ -15,9 +15,45 @@ the appropriate section of the docs.
...
+  CrateDB to this version. This can be done by exporting the data with ``COPY
+  TO`` and importing it into a new table with ``COPY FROM``.  Alternatively you
+  can use ``INSERT`` with query.
```
Comment (mxm) [inline CHANGES.txt]: Should we explain links to explain the two variants in more detail?

File: CHANGES.txt [hunk 4]
```diff
@@ -15,9 +15,45 @@ the appropriate section of the docs.
...
+- The ``error_trace`` GET parameter of the HTTP endpoint only allows ``true``
+  and ``false`` in lower case. Other values are not allowed any more and will
+  result in a parsing exception.
+
+- Store level throttling settings (``indices.store.throttle.*``) have been
+  removed.
+
+- The gateway recovery table setting ``recovery.initial_shards`` has been
+  removed. Nodes will recover their unassigned local primary shards
+  imediatelly after restart.
```
Comment (mxm) [inline CHANGES.txt]: immediately

File: CHANGES.txt (hunk 3 above)
Comment (chaudum) [reply CHANGES.txt]: We could do that in upgrade notes for the release.

**Pull request merged**